app/energy_agent/smart_meter_reader/modbus_reader.py

```diff
@@ -0,0 +1,86 @@
+from pymodbus.client import ModbusSerialClient
+import time
+import binascii
+import re
+from app.helpers.logs import logger, log
+
+# Configuration
+SERIAL_PORT = "/dev/ttyUSB0"
```

Comment:
does this potentially collide with the trust wallet? shouldn't we make this configurable?

Reply:
This is just the default config. 

i removed the constant and added the default parameter to the initalisation of that Reader, 

The actual config would come from the SmartMeterConfig loaded before startup. Than can be defined on the energy agent webpage under additional options smartmeter